module/validation/seal_validator.go

```diff
@@ -44,26 +36,27 @@ func NewSealValidator(
 	results storage.ExecutionResults,
 	seals storage.Seals,
 	assigner module.ChunkAssigner,
-	requiredApprovalsForSealConstruction uint,
+	requiredApprovalsForSealConstructionGetter module.RequiredApprovalsForSealConstructionInstanceGetter,
 	requiredApprovalsForSealVerification uint,
 	metrics module.ConsensusMetrics,
 ) (*sealValidator, error) {
+	requiredApprovalsForSealConstruction := requiredApprovalsForSealConstructionGetter.GetValue()
 	if requiredApprovalsForSealConstruction < requiredApprovalsForSealVerification {
 		return nil, fmt.Errorf("required number of approvals for seal construction (%d) cannot be smaller than for seal verification (%d)",
 			requiredApprovalsForSealConstruction, requiredApprovalsForSealVerification)
 	}
```

Comment:
we have now implemented the validation logic redundantly in different places. From my perspective, it would be beneficial to re-use the same implementation. Thereby we reduce the risk of inconsistent code updates.

Reply:
The above validation is now just a sanity check. The validation has been moved to module/updatable_configs/validation/require_approvals_for_seal_construction.go